src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos: todos }) => {
```

Comment:
The destructuring assignment `{ todos: todos }` can be simplified to `{ todos }` since the property name and the variable name are the same.